code-challenges.js

```diff
@@ -44,6 +84,26 @@ var stringWithVowels2 = "ILoveJavaScript"
 
 var notAString1 = true
 var notAString2 = 42
+var stringWithVowels1 = "HeyThereLearnStudent"
+var stringWithVowels2 = "ILoveJavaScript"
+
+ // declare a function that takes in an argument of a string
+  const noVowels = (str) => {
+  //write an if els statement that informs the user if they have passed in an argument thats not a string
+      if(typeof(str) === "number"){
+        return "function is not a string"
+      }else if(typeof(str) === "boolean"){
+        return "function is not a string"
+      } else  {
+    // next add variable to create vowel removal function
+      let array = str.split("")
+      let vowels = ["a","e","i","o","u","A","E","I","O","U"]
+    //create a higher order function returning the sentence with no noVowels
+      return array.filter(value => vowels.indexOf(value) === -1).join("")
+    }
+ }
+ //call the function and run it in the terminal
+ console.log(noVowels(stringWithVowels2))
```

Comment:
nice approach! you could probably save yourself some key strokes by saying `if(typeof str !== "string")` instead but this solution runs perfectly either way!